java/vector/pom.xml

```diff
@@ -51,15 +53,10 @@
       <artifactId>commons-codec</artifactId>
       <version>1.15</version>
     </dependency>
-    <dependency>
-      <groupId>org.apache.arrow</groupId>
-      <artifactId>arrow-memory-netty</artifactId>
-      <scope>test</scope>
-    </dependency>
```

Comment:
I think should be added back

Reply:
But if I am correct, we no longer need `memory-netty` dependency?

Reply:
The changes have been reverted in https://github.com/sirensolutions/arrow/pull/50/commits/fb7d4277d4f61340b3674c375dbf40e00daf7b5e